### PR TITLE
feat: Release notes for Codacy Cloud December 2023 DOCS-448

### PR DESCRIPTION
**Pull request by claudiacarpinteiro**
Adds auto-generated release notes for Codacy Cloud December 2023

### :eyes: Live preview
https://feat-release-notes-cloud-2023-12--docs-codacy.netlify.app/release-notes/cloud/cloud-2023-12/

### :construction: To do
- [x] Add the new release notes page to `mkdocs.yml` and `docs/release-notes/index.md`
- [x] Review generated release notes
- [x] Review list of issues with missing release notes
- [x] Review links to changelogs of updated tools
- [x] Update release date and remove `TODO` comments

### CODE REVIEW

**Review by nicklem — APPROVED** [review]
:shipit:

**Comment by machadoit** [comment]
Coverage wise they LGTM! 👌 

One thing that may be interesting to share, that maybe it was not very visible:

* Improvements on the coverage reporter that now tries rely on git information to improve the generation of filenames on the coverage reporter (scope: https://codacy.atlassian.net/browse/ALA-792) (this one was probably not given enough visibility, and not synced with @nicklem, my bad. For any further details / discussion, you can ping me / @pedrobpereira )

Lastly, it may be worth to mention https://docs.codacy.com/release-notes/cloud/cloud-2023-11-23-new-coverage-engine-status-checks/ as a 'know more about the new engine rollout plan'? As we there have an up-to-date table and relevant information

**Review by mrfyda — APPROVED** [review]

**Comment by claudiacarpinteiro** [comment]
> One thing that may be interesting to share, that maybe it was not very visible:
> 
> - Improvements on the coverage reporter that now tries rely on git information to improve the generation of filenames on the coverage reporter (scope: https://codacy.atlassian.net/browse/ALA-792) (this one was probably not given enough visibility, and not synced with @nicklem, my bad. For any further details / discussion, you can ping me / @pedrobpereira )

@machadoit I added it as ALA-716 (the Epic).
Do we need to provide any further instructions to users to accommodate this change?

File: docs/release-notes/cloud/cloud-2023-12.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+# Cloud December 2023
+
+These release notes are for the Codacy Cloud updates during December 2023.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+
+-   The [Security and risk management feature](../../organizations/managing-security-and-risk.md), previously only available to organization admins and security managers, is now open to all organization members and shows information related to the repositories that each member follows. (TAROT-2442)
+-   The [Codacy Coverage Reporter](https://coverage.codacy.com/) now only allows reports with filenames matching the files on your Git provider. (ALA-716)
```
Comment (nicklem) [inline docs/release-notes/cloud/cloud-2023-12.md:19]: * I think we can link to our release note here.
* I updated this to sound less "threatening" 😄 

```suggestion
-   When calculating coverage data, the [new coverage engine](./cloud-2023-11-23-new-coverage-engine-status-checks.md) now ignores files in the coverage report that are not in the associated Git repository. (ALA-716)
```

File: docs/release-notes/cloud/cloud-2023-12.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+# Cloud December 2023
+
+These release notes are for the Codacy Cloud updates during December 2023.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+
+-   The [Security and risk management feature](../../organizations/managing-security-and-risk.md), previously only available to organization admins and security managers, is now open to all organization members and shows information related to the repositories that each member follows. (TAROT-2442)
+-   When calculating coverage data, the [new coverage engine](./cloud-2023-11-23-new-coverage-engine-status-checks.md) now ignores files in the coverage report that are not in the associated Git repository. (ALA-716)
```
Comment (claudiacarpinteiro) [inline docs/release-notes/cloud/cloud-2023-12.md:19]:
```suggestion
-   When calculating coverage data, the [new coverage engine](./cloud-2023-11-23-new-coverage-engine-status-checks.md) now ignores files in the coverage report that aren't in the associated Git repository. (ALA-716)
```

**Comment by claudiacarpinteiro** [comment]
@machadoit Ready for a final review. 🙏

**Pull request merged**